tests/results.cpp

```diff
@@ -320,7 +320,7 @@ TEST_CASE("results: notifications") {
     }
 
     // Sort in descending order
-    results = results.sort({{0}, {false}});
+    results = results.sort({*table, {{0}}, {false}});
```

Comment:
It's definitely gross in the tests, but obj-c was using the table when constructing SortDescriptors anyway, so a convenience wrapper wouldn't help it. Might be worth adding something for the other bindings, though.